molecule/openstack/create.yml

```
@@ -2,128 +2,6 @@
-        remote_ip_prefix: "{{ item.cidr | default(omit) }}"
-        remote_group: "{{ item.group | default(omit) }}"
-        ethertype: "{{ item.ethertype | default(omit) }}"
-      with_items: "{{ security_group_rules }}"
-
-    - name: Test for presence of local keypair
-      stat:
-        path: "{{ keypair_path }}"
-      register: keypair_local
-
-    - name: Delete remote keypair
-      os_keypair:
-        name: "{{ keypair_name }}"
-        state: absent
-      when: not keypair_local.stat.exists
-
-    - name: Create keypair
-      os_keypair:
-        name: "{{ keypair_name }}"
-      register: keypair
-
-    - name: Persist the keypair
-      copy:
-        dest: "{{ keypair_path }}"
-        content: "{{ keypair.key.private_key }}"
-        mode: 0600
-      when: keypair.changed
-
-    - name: Create molecule instance(s)
-      os_server:
-        name: "{{ item.name }}"
-        image: "{{ item.image }}"
-        flavor: "{{ item.flavor }}"
-        security_groups: "{{ security_group_name }}"
-        key_name: "{{ keypair_name }}"
-        delete_fip: true
-      register: server
-      with_items: "{{ molecule_yml.platforms }}"
-      async: 7200
-      poll: 0
-
-    - name: Wait for instance(s) creation to complete
-      async_status:
-        jid: "{{ item.ansible_job_id }}"
-      register: os_jobs
-      until: os_jobs.finished
-      retries: 300
-      with_items: "{{ server.results }}"
-
-    # Mandatory configuration for Molecule to function.
-
-    - name: Populate instance config dict
-      set_fact:
-        instance_conf_dict:
-          instance: "{{ item.openstack.name }}"
-          address: "{{ item.openstack.accessIPv4 }}"
-          user: "{{ ssh_user }}"
-          port: "{{ ssh_port }}"
-          identity_file: "{{ keypair_path }}"
-      with_items: "{{ os_jobs.results }}"
-      register: instance_config_dict
-      when: server.changed | bool
-
-    - name: Convert instance config dict to a list
-      set_fact:
-        instance_conf: "{{ instance_config_dict.results \
-                        | map(attribute='ansible_facts.instance_conf_dict') \
-                        | list }}"
-      when: server.changed | bool
-
-    - name: Dump instance config
-      copy:
-        # NOTE(retr0h): Workaround for Ansible 2.2.
-        #               https://github.com/ansible/ansible/issues/20885
-        content: "{{ instance_conf | to_json | from_json | molecule_to_yaml \
-                  | molecule_header }}"
-        dest: "{{ molecule_instance_config }}"
-      when: server.changed | bool
-
-    - name: Wait for SSH
-      wait_for:
-        port: "{{ ssh_port }}"
-        host: "{{ item.address }}"
-        search_regex: SSH
-        delay: 10
-      with_items: "{{ lookup('file', molecule_instance_config) \
-                   | molecule_from_yaml }}"
+  roles:
+    - role: oasis_roles.molecule_openstack_ci
+      molecule_openstack_ci_state: absent
```

github ate my last comment, so apologies if this is redundant...

`requirements.yml` needs to include the ci role.

```suggestion
      molecule_openstack_ci_state: present
```

...or just delete the line.

---

Oops!